src/Sylius/Component/Customer/Model/Customer.php

```diff
@@ -266,6 +271,22 @@ public function setPhoneNumber($phoneNumber)
         $this->phoneNumber = $phoneNumber;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function isNewsletterSubscription()
```

Comment:
isSubscribedToNewsletter? (we can skip "The")
protected $subscribedToNewsletter = false;